BrainPortal/app/models/admin_user.rb

```diff
@@ -36,6 +36,12 @@ def viewable_groups
     Group.where(nil)
   end
 
+  # List of groups that the user can list in the interface. Normally, groups that are invisible
+  # are not listed
```

Comment:
The comment: `Normally, groups that are invisible are not listed` is misleading here since it is not the case for the admin.